In [ ]:
import pandas

In [1]:
my_zoo = pandas.read_csv("adopce-zvirat.csv", sep=";")
my_zoo.tail(35)

NameError: name 'pandas' is not defined

In [3]:
my_zoo.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 513 entries, 0 to 512
Data columns (total 6 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   id           513 non-null    int64 
 1   nazev_cz     513 non-null    object
 2   nazev_en     513 non-null    object
 3   trida_cz     513 non-null    object
 4   cena         513 non-null    int64 
 5   k_prohlidce  513 non-null    int64 
dtypes: int64(3), object(3)
memory usage: 24.2+ KB


soubor má: 
512 řádků, 6 sloupců - id, název_cz, název_en, trida_cz.,  cena, k_prohlidce 

In [4]:
animal_number_34 = my_zoo.iloc[34]
animal_number_34[["nazev_cz","nazev_en"]]

nazev_cz     Ibis bílý
nazev_en    White ibis
Name: 34, dtype: object

Zvíře na řádku č. 34: 
nazev_cz     Ibis bílý
nazev_en    White ibis
Name: 34, dtype: object

Bonus: Využij metody sort_values() (a libovolných dalších metod), ke zjištění následujících informací:

    Adopce kterých zvířat je nejdražší?
    Které zvíře je alfabeticky poslední v češtině? Které v angličtině?

Bonus můžeš doplnit o nějaké vlastní pozorování.

In [5]:
animal_price = my_zoo.sort_values(by="cena")
animal_price[["nazev_cz","cena"]].tail(10)
                  

nazev_cz   cena
115                                Lev indický  30000
105                               Medvěd lední  30000
102                             Gorila nížinná  30000
244                             Varan komodský  30000
114                            Tygr sumaterský  50000
113                            Tygr ussurijský  50000
272                              Tygr malajský  50000
129                          Hroch obojživelný  50000
135  Žirafa severní núbijská (= Rothschildova)  50000
128                               Slon indický  50000

nejdražší zvířata k adopci jsou: 
114	Tygr sumaterský	50000
113	Tygr ussurijský	50000
272	Tygr malajský	50000
129	Hroch obojživelný	50000
135	Žirafa severní núbijská (= Rothschildova)	50000
128	Slon indický	50000

In [6]:
animal_name=my_zoo.sort_values(by="nazev_cz")
animal_name["nazev_cz"].tail(1)

371    Žluva hajní
Name: nazev_cz, dtype: object

In [10]:
animal_name_EN=my_zoo.sort_values(by="nazev_en")
animal_name_EN["nazev_en"].tail(1)

161    Yellow-necked francolin
Name: nazev_en, dtype: object

In [14]:
#průměrná cena  adopce 
price_ave = (my_zoo["cena"].mean()).round(0)
price_ave

3310.0